x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/post_comment.ts

```diff
@@ -272,6 +272,24 @@ export default ({ getService }: FtrProviderContext): void => {
             expectedHttpCode: 400,
           });
         });
+
+        it('400s when attempting to add more than 100 files to a case in a single attachment', async () => {
```

Comment:
I noticed I forgot to add a test where we have a bunch of files in a single attachment (since the metadata files field is an array).